From 22552b8e95b765e43de793f2cc49bba1e24ceeb6 Mon Sep 17 00:00:00 2001
From: Matti Kosola <matti.kosola@jolla.com>
Date: Wed, 12 Aug 2020 16:44:57 +0300
Subject: [PATCH] Remove android define from logging.

Decide if logging is enabled or disabled based on RELEASE_OR_BETA
define. ANDROID define should not be never enabled on sfos based
devices.

Signed-off-by: Matti Kosola <matti.kosola@jolla.com>
---
 xpcom/base/Logging.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xpcom/base/Logging.h b/xpcom/base/Logging.h
index 040fb9c49720..482e2ab61022 100644
--- a/xpcom/base/Logging.h
+++ b/xpcom/base/Logging.h
@@ -31,7 +31,7 @@
 // devices, we've chosen to leave logging enabled on desktop, but disabled on
 // Android.  Given that logging can still be useful for development purposes,
 // however, we leave logging enabled on Android developer builds.
-#if !defined(ANDROID) || !defined(RELEASE_OR_BETA)
+#if !defined(RELEASE_OR_BETA)
 #define MOZ_LOGGING_ENABLED 1
 #else
 #define MOZ_LOGGING_ENABLED 0
-- 
2.26.2

